src/core.js

```diff
@@ -30,7 +30,7 @@ define([
     'jquery.jstree',
     'jquery.bootstrap',
     'jquery.fancybox',  // only thing we use fancybox for is its spinner, no actual display of anything
-    'jquery-ui',  // used for buttons in Edit Source XML, and dialogs
+    'jquery-ui',  // used for autocomplete
```

Comment:
Hopefully we can replace autocomplete with bootstrap's "typeahead" - haven't looked into it yet, but planning on it.